fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -77,7 +77,7 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
         webExtensionPromptFeature.set(
             feature = WebExtensionPromptFeature(
                 store = requireComponents.core.store,
-                provideAddons = { addons },
+                provideAddons = { if (::addons.isInitialized) addons else emptyList() },
```

Comment:
This prevent accessing `addons` before is ready. This looks like an edge case that I'm not completely aware of how could it happens as the functionality is completely driven by users, the `addons` list should be ready for users to be able to interact with the UI that triggers the call on `provideAddons`, it may be the case that we getting stuck with an old request, providing an empty list will prevent the crash and remove the stale request if any  [see](https://github.com/mozilla-mobile/firefox-android/blob/ce046588f005ae84d78cfb6e5b2c7c06618dfcc2/fenix/app/src/main/java/org/mozilla/fenix/extension/WebExtensionPromptFeature.kt#L55-L75).